Network/HTTP/Req.hs

```diff
@@ -1286,6 +1287,17 @@ basicAuthUnsafe
 basicAuthUnsafe username password = asFinalizer
   (pure . L.applyBasicAuth username password)
 
+-- | The 'Option' adds basic Proxy authentication
+--
+-- @since 1.0.1
```

Comment:
@mrkkrp After checking the outstanding pr, If the rest is Ok this should move to 1.1.0 right ?

Reply:
Right, 1.1.0.